packages/js/src/utils/scopes.test.ts

```diff
@@ -0,0 +1,38 @@
+import { withReservedScopes } from './scopes';
+
+const email = 'email';
+const name = 'name';
+const offlineAccess = 'offline_access';
+const openid = 'openid';
```

Comment:
nit: `const openId = 'open_id'`?

Reply:
@demonzoo `openid` is reserved scope in oidc, please check rfc

Reply:
ok, my bad